docs/changelog.md

```diff
@@ -40,6 +40,38 @@ The versions of JanusGraph listed below are outdated and will no longer receive
 
 ## Release Notes
 
+### Version 0.5.3 (Release Date: December 24, 2020)
+
+=== "Maven"
+    ```xml
+    <dependency>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph-core</artifactId>
+        <version>0.5.3</version>
+    </dependency>
+    ```
+
+=== "Gradle"
+    ```groovy
+    compile "org.janusgraph:janusgraph-core:0.5.3"
+    ```
+
+**Tested Compatibility:**
+
+* Apache Cassandra 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 1.2.6, 1.3.1, 1.4.10, 2.1.5
+* Google Bigtable 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.11.0, 1.14.0
```

Comment:
I asked once in https://github.com/JanusGraph/janusgraph/discussions/2299, but since we are here, let me paste my question here:

How is Google BigTable compatibility tested given that (AFAIK) it is not open-sourced and only available on Google Cloud Platform?

Reply:
I think @mbrukman can answer that better. 
As far as I know it was tested manually.